python/tskit/trees.py

```diff
@@ -2970,13 +2970,16 @@ def rf_distance(self, other):
 
     def path_length(self, u, v):
         """
-        Returns the path length between two nodes
-        (i.e., the number of edges between two nodes in this tree).
+        Returns a count of the number of edges between two nodes in this tree.
```

Comment:
The "a count" is redundant here, right (saying it implies that there's more than one way to count the number of edges traversed). I think saying "path" somewhere in the description is important, as it's talking about the path traversed when walking the tree from one node to the other. Just saying "between" could be confused with counting all edges in the subtree rooted at the MRCA or something.

Reply:
At the moment it says "Returns the path length between two nodes", which I immediately interpret as a branch length. I am trying hard to point out that this is an integer count, not (really) a length in the normal tree-parlance.

I guess you could say "Counts the number of edges when walking the path between two nodes in the tree"?

Reply:
```suggestion
        Returns the number of edges on the path in this tree between the two nodes.
```

Reply:
LGTM